roles/ceph-osd/defaults/main.yml

```diff
@@ -0,0 +1,102 @@
+---
+# You can override default vars defined in defaults/main.yml here,
+# but I would advice to use host or group vars instead
+
+####################
+# OSD CRUSH LOCATION
+####################
+
+# The following options will build a ceph.conf with OSD sections
+# Example:
+# [osd.X]
+# osd crush location = "root=location"
+#
+# This works with your inventory file
+# To match the following 'osd_crush_location' option the inventory must look like:
+#
+# [osds]
+# osd0 ceph_crush_root=foo ceph_crush_rack=bar
+
+crush_location: false
+osd_crush_location: "'root={{ ceph_crush_root }} rack={{ ceph_crush_rack }} host={{ ansible_hostname }}'"
+
+##############
+# CEPH OPTIONS
+##############
+
+# ACTIVATE THE FSID VARIABLE FOR NON-VAGRANT DEPLOYMENT
+fsid: "{{ cluster_uuid.stdout }}"
+cephx: true

```

Comment:
These options that are similarly used in other roles should just be defined in ceph-common, since all other ceph roles should depend on the ceph-common role. We want our variables to be defined once.